### PR TITLE
Update tests to include 1 for checking if the console works

### PR DESCRIPTION
**Pull request by ryestew**

### CODE REVIEW

**Comment by ryestew** [comment]
This PR has 3 returns in it - it really needed 2 to work.  But as soon as the terminal is fixed it will only need 1 return.

File: test-browser/tests/console.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+module.exports = {
+  before: function (browser, done) {
+    init(browser, done)
+  },
+  'SimpleExecutionConsole': function (browser) {
+    browser
+    .waitForElementVisible('#terminalCli', 10000)
+    .executeScript(['\uE01B','\uE025','\uE01B','\uE006','\uE006','\uE006'])
+    .waitForElementVisible('#journal div:last-child span.text-info', 10000)
```
Comment (yann300) [inline test-browser/tests/console.js]: `waitForElementVisible` should perhaps be in `journalLastChild` (this is the thing you are waiting to appear)

File: test-browser/tests/console.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+var sauce = require('./sauce')
+
+module.exports = {
+  before: function (browser, done) {
+    init(browser, done)
+  },
+  'SimpleExecutionConsole': function (browser) {
+    browser
+    .waitForElementVisible('#terminalCli', 10000)
+    .executeScript(['\uE01B','\uE025','\uE01B','\uE006','\uE006','\uE006'])
```
Comment (yann300) [inline test-browser/tests/console.js]: does that include hitting `return`?

File: test-browser/commands/journalLastChild.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const EventEmitter = require('events')
+
+class JournalLastChild extends EventEmitter {
+  command (val) {
+    this.api.waitForElementVisible('#journal div:last-child span.text-info', 10000)
```
Comment (yann300) [inline test-browser/commands/journalLastChild.js]: I think you have to queue the calls here

Comment (yann300) [inline test-browser/commands/journalLastChild.js]: I think you have to queue the call here

File: test-browser/tests/console.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+var sauce = require('./sauce')
+
+module.exports = {
+  before: function (browser, done) {
+    init(browser, done)
+  },
+  'SimpleExecutionConsole': function (browser) {
+    browser
+    .waitForElementVisible('#terminalCli', 10000)
+    .executeScript(['1+1'])
```
Comment (yann300) [inline test-browser/tests/console.js]: this should work without the input being an array

File: test-browser/commands/journalLastChild.js (hunk 3 above)
Comment (GrandSchtroumpf) [inline test-browser/commands/journalLastChild.js]: Maybe use only ids

**Pull request merged**